modules/standard/Filerator.chpl

```diff
@@ -32,12 +32,12 @@ use Sort;
 // wrong.
 //
 
```

Comment:
I'm sure it's on your documentation list, but the comments above are outdated now